src/main/java/uk/gov/mca/beacons/api/services/DeleteBeaconService.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.mca.beacons.api.services;
+
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.mca.beacons.api.gateways.BeaconGateway;
+
+@Service
+@Transactional
+public class DeleteBeaconService {
```

Comment:
@tm1303  Went for a separate service here as it's going to also have to capture the note that is recorded based on @Ninamma work for the note work